### PR TITLE
[Revamp Landing Screen] Jetpack App - Set fixed font size for animated text

### LINKED ISSUE #17299: [Revamp Landing Screen] JPAndroid - Font size of looping text should be fixed
Font size of the animated large text should be fixed and not increase/decrease based on font size Accessibility settings.

**Repro**

1. Make sure `LandingScreenRevamp` feature flag is enabled

<details>
<summary>How to Enable the <code>LandingScreenRevamp</code> feature flag</summary>

- Either `login` → `Me` → `App Settings` → `Debug Settings` → enable `LandingScreenRevamp` → `RESTART THE APP` → `logout` or
- Apply this hard-coded solution:

<details>
<summary>Patch to enable the <code>LANDING_SCREEN_REVAMP</code> build flag</summary>

```diff
diff --git a/WordPress/build.gradle b/WordPress/build.gradle
index 63260a4ca8..fe999fa92f 100644
--- a/WordPress/build.gradle
+++ b/WordPress/build.gradle
@@ -112,7 +112,7 @@ android {
         buildConfigField "boolean", "READER_COMMENTS_MODERATION", "false"
         buildConfigField "boolean", "SITE_INTENT_QUESTION", "true"
         buildConfigField "boolean", "SITE_NAME", "false"
-        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "false"
+        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "true"
         buildConfigField "boolean", "LAND_ON_THE_EDITOR", "false"
         buildConfigField "boolean", "BLOGGING_PROMPTS", "false"
         buildConfigField "boolean", "QUICK_START_EXISTING_USERS_V2", "false"
```  
</details>

---

</details>

2. Launch Jetpack App and make sure you're logged out
3. Set font size to max large / small via Accessibility settings on your device
4. **Notice** The font size of the animated text is also increased / decreased

### PR DESCRIPTION
**Pull request by ovitrif**
Fixes #17299

To test:
<details>
<summary><b>Prerequisite</b> - Enable the <code>LandingScreenRevamp</code> feature flag</summary>

- Either `login` → `Me` → `App Settings` → `Debug Settings` → enable `LandingScreenRevamp` → `RESTART THE APP` → `logout` or
- Apply this hard-coded solution:

<details>
<summary>Patch to enable the <code>LANDING_SCREEN_REVAMP</code> build flag</summary>

```diff
diff --git a/WordPress/build.gradle b/WordPress/build.gradle
index 63260a4ca8..fe999fa92f 100644
--- a/WordPress/build.gradle
+++ b/WordPress/build.gradle
@@ -112,7 +112,7 @@ android {
         buildConfigField "boolean", "READER_COMMENTS_MODERATION", "false"
         buildConfigField "boolean", "SITE_INTENT_QUESTION", "true"
         buildConfigField "boolean", "SITE_NAME", "false"
-        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "false"
+        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "true"
         buildConfigField "boolean", "LAND_ON_THE_EDITOR", "false"
         buildConfigField "boolean", "BLOGGING_PROMPTS", "false"
         buildConfigField "boolean", "QUICK_START_EXISTING_USERS_V2", "false"
```  
</details>

---

</details>

1. Launch Jetpack App and make sure you're logged out
2. Set font size to max large / small via Accessibility settings on your device
3. **Expect** The font size of the animated text to **not** change

## Previews

| Smallest Font Size | Largest Font Size |
| - | - |
|  |  |

## Regression Notes
1. Potential unintended areas of impact
   N/a

3. What I did to test those areas of impact (or what existing automated tests I relied on)
   N/a

4. What automated tests I added (or what prevented me from doing so)
   N/a

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/jetpack/java/org/wordpress/android/ui/accounts/login/components/LoopingText.kt [hunk 1]
```diff
@@ -20,11 +21,13 @@ import org.wordpress.android.ui.accounts.login.LocalPosition
...
 import org.wordpress.android.util.extensions.isOdd
 
-private val fontSize = 40.sp
-private val lineHeight = fontSize / 100 * 105 // last value = % of fontSize
+private const val FIXED_FONT_SIZE = 40
 
 @Composable
 private fun LargeTexts() {
+    val fontSize = (FIXED_FONT_SIZE / LocalDensity.current.fontScale).sp
+    val lineHeight = fontSize / 100 * 105 // last value = % of fontSize
```
Comment (mkevins) [inline WordPress/src/jetpack/java/org/wordpress/android/ui/accounts/login/components/LoopingText.kt]: Very minor nitpick:

The order of operations (and comment) are slightly confusing (I know the precedence, but at first glance it almost seems like 105 could be in the denominator).

I was thinking maybe we could change this to:

```suggestion
    val lineHeight = fontSize * 1.05 // set line height to 5% larger than the font size
```

Wdyt?

**Review by mkevins — APPROVED** [review]
This works well in my testing (on a Pixel 3a, physical device). I first reproduced the issue on `trunk`, and I verified that this fixes the issue. Nice work Ovi!

I left a very minor nitpick comment / suggestion for consideration, but nothing blocking. Feel free to merge either way.

File: WordPress/src/jetpack/java/org/wordpress/android/ui/accounts/login/components/LoopingText.kt (hunk 1 above)
Comment (ovitrif) [reply WordPress/src/jetpack/java/org/wordpress/android/ui/accounts/login/components/LoopingText.kt]: > The order of operations (and comment) are slightly confusing (I know the precedence, but at first glance it almost seems like 105 could be in the denominator).

Totally agree @mkevins, I've been thinking of updating this line of code each time I looked at it 😅 .

Thank you for suggesting the improvement 🙇

**Comment by ovitrif** [comment]
I ran the CI Checks a few more times since the `instrumented-tests` was failing although unrelated to the latest change.

Now the `installable-builds` keep failing, the reason being that there is already a build created with that name.

I expect this to not be a blocker for merging 🤞 .

**Pull request merged**